blob/helper.go

```diff
@@ -35,6 +46,29 @@ func sharesToBlobs(rawShares [][]byte) ([]*Blob, error) {
 	return blobs, nil
 }
 
+// BlobsToShares accepts blobs and convert them to the Shares.
+func BlobsToShares(blobs ...*Blob) ([]share.Share, error) {
+	b := make([]types.Blob, 0)
+	for _, blob := range blobs {
+		b = append(b, types.Blob{
+			NamespaceID:  blob.NamespaceID(),
+			Data:         blob.Data(),
+			ShareVersion: uint8(blob.Version()),
+		})
+	}
+
+	sort.Slice(b, func(i, j int) bool {
+		val := bytes.Compare(b[i].NamespaceID, b[j].NamespaceID)
+		return val <= 0
+	})
+
+	rawShares, err := shares.SplitBlobs(0, nil, b, false)
```

Comment:
our blobs should go in increasing order of nID otherwise we will get an error if will try to build an nmt tree from `rawShares`. Sorting protects from this error

Reply:
OK